lib/screens/explore_screens/invite/interactor/mappers/seeds_amount_change_mapper.dart

```diff
@@ -1,9 +1,9 @@
 import 'package:seeds/blocs/rates/viewmodels/rates_state.dart';
+import 'package:seeds/datasource/local/models/token_data_model.dart';
 import 'package:seeds/datasource/local/settings_storage.dart';
 import 'package:seeds/domain-shared/result_to_state_mapper.dart';
 import 'package:seeds/screens/explore_screens/invite/interactor/viewmodels/invite_state.dart';
 import 'package:seeds/utils/rate_states_extensions.dart';
-import 'package:seeds/utils/double_extension.dart';
 import 'package:seeds/i18n/explore_screens/invite/invite.i18n.dart';
 
 class SeedsAmountChangeMapper extends StateMapper {
```

Comment:
This class name doesnt really represent what is going on here right? 
Is not a seeds amount anymore, now its TokenAmount

Reply:
Rigth
from `SeedsAmountChangeMapper `to `TokensAmountChangeMapper`

Reply:
Ok but that adds lines!! 

Will add separate PR to rename it

Reply:
Actually never mind, this one is hard coded on Seeds so the name can stay as it is

It's using a SeedsToken data model - this is all as it should be.

Reply:
I think the name of this class is weird, since there's a few SeedsAmountChangeMapper classes running around the code, but they're for different use cases - this one is for invites. Invites are always in Seeds. 

I would be all for cleaning up these names, seeing if we can combine these things intelligently, or rename them so we don't have duplicated names in the code

But not in this PR!